components/top/OurPrograms.vue

```diff
@@ -2,36 +2,104 @@
   <div class="programs-container">
     <h2>Our Programs</h2>
     <div class="programs-contents">
-      <img src="https://4.bp.blogspot.com/-aJjlevJyK9U/XGjx40QThrI/AAAAAAABRco/j9aRPnYHpX4PU6RZjhTWRh6_8xnPfbNEQCLcBGAs/s400/animal_chara_computer_azarashi.png" alt="">
-      <img src="https://1.bp.blogspot.com/-aqFVQ75FKAY/XQjuJWzCfzI/AAAAAAABTPk/Mxb0ORXi96sYBcF7WCqTQyZlcQNI5JZ5ACLcBGAs/s400/jiko_computer_coffee_kobosu.png" alt="">
-      <img src="https://3.bp.blogspot.com/-I5omlyOiw04/W1vg2o3hHgI/AAAAAAABNr4/_PpEM8ZROX42aTaNNZQK2_p2JT7Bfp8zgCLcBGAs/s400/animal_chara_computer_neko.png" alt="">
+
+      <div class="programs-contents-card global-talent">
+        <h3>Global Talent</h3>
+        <p class="programs-contents-card-text">海外のビジネス界に飛び込む <br>インターンシッププログラム。 <br>海外の企業で力を伸ばし、<br> 世界を、自分を、前進させる経験を。
+        </p>
+        <p class="programs-contents-card-link">学生の方は <a href="/outgoing">こちらから</a></p>
+        <p class="programs-contents-card-link">企業の方は <a href="/incoming">こちらから</a></p>
+      </div>
+
+      <div class="programs-contents-card global-volunteer">
+        <h3>Global Volunteer</h3>
+        <p class="programs-contents-card-text">海外の社会課題解決に挑戦する <br>ボランティアプログラム。 <br>世界を見て、考えて、行動する。<br> 世界に挑戦するための一歩目を。
+        </p>
+        <p class="programs-contents-card-link">参加については <a href="/outgoing">こちらから</a></p>
+      </div>
+
+      <div class="programs-contents-card online-events">
+        <h3>Online Events</h3>
+        <p class="programs-contents-card-text">世界を身近に感じることのできる<br> オンラインイベント。 <br>国内外の様々な話を聞いて、 <br>知らなかった自分や世界を見つけよう。
+        </p>
+<!--        FIXME : リンク先あとで入れる -->
+        <p class="programs-contents-card-link">参加については <a href="">こちらから</a></p>
+      </div>
+
     </div>
   </div>
 </template>
 
-<style scoped lang="scss">
-.programs-container {
-  width: 100%;
-  min-height:500px;
-  display: flex;
-  flex-direction: column;
-  text-align: center;
-  h2 {
-    color: $blue;
-    font-size: 24px;
-    letter-spacing: 2px;
-    padding-top: 150px;
+<style lang="scss" scoped>
+.programs {
+  &-container {
+    width: 100%;
+    min-height: 500px;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+
+    h2 {
+      color: $dark-gray;
+      font-size: 24px;
+      letter-spacing: 2px;
+      padding-top: 150px;
+    }
   }
-}
-.programs-contents {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  padding-top: 100px;
-  img {
-    width: 350px;
-    height: 300px;
-    padding: 30px;
+
+  &-contents {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    padding-top: 100px;
+
+    &-card {
+      width: 380px;
+      height: 450px;
+      color: white;
+      font-family: $fonts;
+      font-weight: bold;
+      border-radius: 10px;
```

Comment:
8px入れておきましたが6pxでもよいかもです

```suggestion
    &-card {
      width: 380px;
      height: 450px;
      color: white;
      font-family: $fonts;
      font-weight: bold;
      border-radius: 10px;
      margin: 8px;
```

Reply:
あざす。一旦その提案取り込んで確認してみます。